test/e2e/playbook/provision.yaml

```
@@ -119,27 +124,48 @@
           - tomli_w
         state: present
 
-    # We need quite recent containerd that has NRI support so use the one
-    # that is compiled from sources.
-    - name: copy containerd sources
-      copy: src="{{ item }}" dest="/usr/bin" mode=0755
+    # Install containerd from a release tarball to /usr/local.
+    - name: Install containerd binaries from release tarball
+      ansible.builtin.unarchive:
+        src: "https://github.com/containerd/containerd/releases/download/v{{ containerd_release }}/containerd-{{ containerd_release }}-linux-amd64.tar.gz"
+        dest: /usr/local
+        remote_src: yes
+      when: is_containerd and containerd_src == ""
+
```

```suggestion
        remote_src: yes
      when: 
        - is_containerd
        - not containerd_src
```
maybe something like this would be easier? `not containerd_src` equals to` if containerd_src != true`

---

There is no need for the `is_{containerd,crio}_src` variables, so they have been removed altogether.

---

Actually the is_crio and is_containerd variables are also quite bogus. I'm contemplating removing them and replacing them with the original test they now simply alias: `cri_runtime == "crio"` and `cri_runtime == "containerd"`...